internal/mediainfo/matroska_eac3_probe_test.go

```
@@ -72,3 +76,128 @@ func TestProbeMatroskaAudio_EAC3MultiFramePacket(t *testing.T) {
+		}
+	}
+	if got := stream.JSON["BitRate"]; got != "1536000" {
+		t.Fatalf("JSON BitRate=%q, want 1536000", got)
+	}
+	if got := stream.JSON["Format"]; got != "E-AC-3" {
+		t.Fatalf("JSON Format=%q, want E-AC-3", got)
+	}
+	if got := stream.JSON["Format_AdditionalFeatures"]; got != "JOC" {
+		t.Fatalf("JSON Format_AdditionalFeatures=%q, want JOC", got)
+	}
+	if got := stream.JSONRaw["extra"]; !bytes.Contains([]byte(got), []byte(`"dmixmod":"Lo/Ro"`)) {
```

golangci-lint (2.12.2)

[error] avoid allocations with strings.Contains

(mirror)